{{ cookiecutter.project_shortname }}/setup.cfg

```diff
@@ -1,4 +1,57 @@
-{% include 'misc/header.py' %}
+{% include "misc/header.py" %}
+
+[metadata]
+name = {{ cookiecutter.project_shortname }}
+version = attr: {{ cookiecutter.package_name }}.__version__
+description = "{{ cookiecutter.description }}"
+long_description = file: README.rst, CHANGES.rst
+keywords = invenio TODO
+license = MIT
+author = {{ cookiecutter.author_name }}
+author_email = {{ cookiecutter.author_email }}
+platforms = any
+url = https://github.com/{{ cookiecutter.github_repo }}
+classifiers =
+    Development Status :: 1 - Planning
+
+[options]
+include_package_data = True
+packages = find:
+python_requires = >=3.7
+zip_safe = False
+install_requires =
+    invenio-base>=1.2.5
+    invenio-i18n>=1.3.1
+
+[options.extras_require]
+tests =
+    invenio-app>=1.3.4,<2.0.0
```

Comment:
moderate: I thought we were only putting an upper cap in `invenio-app-rdm` and in cases where there is breaking changes releases by 3rd parties. Otherwise new modules will come with all invenio dependencies capped.

Reply:
We're notoriously inconsistent regarding the version capping on modules, and I can understand that since there are good pros/cons for either choice. I think the major ones are:

+ Better stability: capping prevents modules from breaking when we release breaking changes (i.e. major bump) on the dependency
- More work: without the tests breaking when a dependency has breaking changes, we'll never find out the impact of a breaking change unless we manually check each module that depends on it

I've found a mix of capped/uncapped across base Invenio and RDM modules. Maybe a general "pattern" is that the higher up in the chain a module is or the more critical its dependencies are, the more likely they're capped:

- [rdm-records](https://github.com/inveniosoftware/invenio-rdm-records/blob/08c420f4745a9979cb864ef33d38fdb7f5096138/setup.cfg#L39-L43)
- [records-resources](https://github.com/inveniosoftware/invenio-records-resources/blob/389a96f8fe22103e7ee5ad3ff19c682402403e46/setup.cfg#L30-L39)

I would say given past situations, capping is a more stable alternative that helps also with overall developer experience.

Reply:
I agree with you in pros/cons. However, I tend to lean more on controlled chaos and cap when we notice breaking changes (true that we should have better mechanisms to do so). I'm okay with merging this as-is. However, since this will affect ALL new modules maybe we should raise the topic to a larger audience (thursday? architects)?

TL;DR: feel free to merge, maybe worth discussing with broader audience.